apps/SageAccountWeb/src/components/ProfileValidation/ProfileValidation.tsx

```diff
@@ -431,7 +425,17 @@ export const ProfileValidation = (props: ProfileValidationProps) => {
   return (
     <StyledOuterContainer>
       {step !== ValidationWizardStep.THANK_YOU ? (
-        <StyledInnerContainer>
+        <StyledInnerContainer
+          sx={
+            step === ValidationWizardStep.TERMS_AGREE
+              ? {
+                  width: '1200px',
+                  '& > div:nth-of-type(1)': { paddingTop: 10, width: '750px' },
+                  '& > div:nth-of-type(2)': { paddingTop: 10 },
```

Comment:
Also would prefer to use the `useTheme` instead of importing the theme directly, but that might be an issue across the app so don't make that change now if it happens to break other things.

Reply:
Agreed, I was confused too! I'll change to theme.spacing and use the `useTheme` hook in these components, it doesn't seem to break anything here.